lib/plugins/repository.js

```diff
@@ -252,4 +256,39 @@ module.exports = class Repository {
       }
     }
   }
-}
+
+  updateAutomatedSecurityFixes(repoData, resArray) {
+    if (this.security) {
+      if (this.security?.enableAutomatedSecurityFixes === true || this.security?.enableAutomatedSecurityFixes === false) {
```

Comment:
This condition is superfluous since you already have for both of them a separate block below.

Reply:
@martinm82 agree 🤔 , as this is my first contribution, I figured I would follow the as the [updateSecurity](https://github.com/github/safe-settings/blob/main-enterprise/lib/plugins/repository.js#LL217C25-L217C25)

Reply:
If you want you can change it as well in the other places but I am not insisting ;)

Reply:
@martinm82 I've updated both functions 😄